libraries/k8s_config_maps.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'k8sobjects'
+
+module Inspec
+  module Resources
+    class K8sConfigMaps < K8sObjects
+      name 'k8s_config_maps'
+      desc 'Verifies settings for all config maps'
+
+      example "
+      describe k8s_config_maps(namespace: 'default') do
+        it { should exist }
+        its('name') { should eq 'my-pod' }
+        ...
+      end
+    "
+      attr_reader :k8sobject
+
+      def initialize(opts = {})
+        Validators.validate_params_required(@__resource_name__, [:namespace], opts)
+        @objtype = 'configmap'
+        super(opts)
+      end
+    end
+  end
+end
+
```

Comment:
Can we add the resource_id and to_s method in k8s_config_map and k8s_config_maps resources?

Reply:
I think the to_s won't be required as the parent class has its implementation but may be the `resource_id`.

Reply:
yeah good point.. `resource_id` will also be part of the parent object.. will make that change